include/sys/fs/zfs.h

```diff
@@ -238,6 +239,7 @@ typedef enum {
 	ZPOOL_PROP_TNAME,
 	ZPOOL_PROP_MAXDNODESIZE,
 	ZPOOL_PROP_MULTIHOST,
+	ZPOOL_PROP_SPECIAL_SMALL_BLOCKS,
```

Comment:
are we going to have dedup and regular metadata be configurable at a pool-level as well? the system-wide parameters in the zfs module are insufficient for my use case

Reply:
@kpande - we will be adding a dedup class which would be configured on a per-pool basis and address your use case.
@ahrens - good point; the proeprty is now a dataset property